README.md

```diff
@@ -204,6 +204,10 @@ For example, see the "lc_subject", "composer", and "rights_holder" fields in [th
 (https://github.com/curationexperts/alexandria-v2/blob/master/spec/fixtures/csv/pamss045_with_local_authorities.csv).
 
 
+# Caveats
+
+* Reindexing all objects (to an empty solr) requires two passes.  The first pass will guarantee that the collections are indexed, and the second pass will index the collection name on all the objects. The object indexer looks up the collection name from solr for speed.
```

Comment:
I'm not sure what this caveat is telling me.  What does "Reindexing all objects" mean?  Are you saying I need to do something like this: 
```
2.times { ActiveFedora::Base.reindex_everything }
```
OR
```
ActiveFedora::Base.each.all do | obj |
    obj.update_index
end
ActiveFedora::Base.each.all do | obj |
    obj.update_index
end
```
OR that I need to index collections first
```
Collection.all.each do | col |
    col.update_index
end
ActiveFedora::Base.reindex_everything
```
Or something else entirely?